test/touch.c

```diff
@@ -23,7 +23,7 @@ static int mtime(char *file)
 	stat(file, &before);
 
 	/* Must sleep a while here otherwise we execute too fast => no mtime change :-( */
-	usleep(10000);
+	sleep(2);
```

Comment:
Test, this is fine :)

Although, in the commit message, could you please elaborate why the number 2 was selected.